app/src/main/kotlin/com/wire/android/ui/home/conversations/model/AttachmentBundle.kt

```diff
@@ -23,6 +30,13 @@ data class AttachmentBundle(val mimeType: String, val rawContent: ByteArray) : P
     override fun hashCode(): Int {
         var result = mimeType.hashCode()
         result = 31 * result + rawContent.contentHashCode()
+        result = 31 * result + fileName.hashCode()
+        result = 31 * result + attachmentType.hashCode()
```

Comment:
quick question: what does `31` means?

Reply:
I think it's just a convention when generating/implementing `hashCode` methods, in general we just have to provide a prime number to be safe =)